src/pickledb_store.rs

```diff
@@ -0,0 +1,76 @@
+use crate::{Location, StoreImpl};
+use pickledb::{PickleDb, PickleDbDumpPolicy, SerializationMethod};
+use serde::{de::DeserializeOwned, Serialize};
+use std::fmt::{Debug, Formatter};
+pub struct PickleDBStore {
```

Comment:
One very minor nitpick: We follow the rust convention that abbreviations are pascal case, so PickleDb.

This is just internal api, so fix it you feel like it :)

Thanks for the contribution!